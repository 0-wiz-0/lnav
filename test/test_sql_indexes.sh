#! /bin/bash

export YES_COLOR=1

run_cap_test ${lnav_test} -n \
    -c ";EXPLAIN QUERY PLAN SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

run_cap_test ${lnav_test} -n \
    -c ";SELECT * FROM syslog_log WHERE log_path GLOB '*/logfile_syslog.*'" \
    ${test_dir}/logfile_syslog.*

