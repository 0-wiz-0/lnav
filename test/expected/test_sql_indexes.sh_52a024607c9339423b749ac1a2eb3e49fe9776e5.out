[1m[7mid [0m[1m[7mparent [0m[1m[7mnotused [0m[1m                                   detail                                     [0m
[1m[7m 2[0m[1m[7m [0m[1m[7m     0[0m[1m[7m [0m[1m[7m      0[0m[1m[7m [0m[1m[7mSCAN syslog_log VIRTUAL TABLE INDEX 1:SEARCH syslog_log USING log_path GLOB[0m[1m ? [0m
